docker-wsl-proxy.go

```diff
@@ -204,3 +212,112 @@ func handleContainerCreate(req *http.Request) error {
 
 	return nil
 }
+
+var containerInspectRegex = regexp.MustCompile("/containers/.*/json")
+
+func proxyModifyResponse(response *http.Response) (err error) {
+	if containerInspectRegex.MatchString(response.Request.RequestURI) && response.StatusCode == http.StatusOK {
+		err = handleContainerInspect(response)
+	}
+
+	return err
+}
+
+func handleContainerInspect(response *http.Response) error {
+	buf := make([]byte, response.ContentLength)
+	_, err := io.ReadFull(response.Body, buf)
+	if err != nil {
+		return err
+	}
+
+	containerJSON := types.ContainerJSON{}
+	err = json.Unmarshal(buf, &containerJSON)
+	if err != nil {
+		return err
+	}
```

Comment:
Обычно подобную хрень пишут чуть по-другому:
```suggestion
	var containerJSON types.ContainerJSON
	if err := json.Unmarshal(buf, &containerJSON); err != nil {
		return err
	}
```